docker/kuberay-autoscaler/autoscaling_config.py

```diff
@@ -0,0 +1,282 @@
+from contextlib import suppress
```

Comment:
with this configuration, we can remove configmap. https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/kuberay/kuberay-autoscaler.yaml#L1

Reply:
Right, thanks for reminding me to `rm` that.